packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -2598,6 +2598,38 @@ QUnit.module('keyboard navigation', {
         assert.ok($cancelButton.hasClass('dx-state-focused'), 'the apply button is focused');
     });
 
+    QUnit.testInActiveWindow('toolbar button should be focused on the "tab" key press if the input is focused and showSelectionControls is enabled', function(assert) {
+        if(devices.real().deviceType !== 'desktop') {
+            assert.ok(true, 'desktop specific test');
+            return;
+        }
+
+        this.reinit({
+            opened: true,
+            showSelectionControls: true,
+            applyValueMode: 'useButtons',
```

Comment:
Does this scenario really specific only to a `showSelectionControls` and `applyValueMode` modes combinations?
I expect it always works this way.

Reply:
We wouldn't see this issue without these two options, so I think test should include them